scapy/contrib/ikev2.py

```diff
@@ -435,16 +435,16 @@ def post_build(self, p, pay):
 
 
 class IKEv2_Key_Length_Attribute(IntField):
-	# We only support the fixed-length Key Length attribute (the only one currently defined)
-	def __init__(self, name):
-		IntField.__init__(self, name, 0x800E0000)
-
-	def i2h(self, pkt, x):
-		return IntField.i2h(self, pkt, x & 0xFFFF)
-		
-	def h2i(self, pkt, x):
-		return IntField.h2i(self, pkt, x if x !=None else 0 | 0x800E0000)
-		
+        # We only support the fixed-length Key Length attribute (the only one currently defined)
+        def __init__(self, name):
+                IntField.__init__(self, name, 0x800E0000)
```

Comment:
It's compulsory to replace tabs by space, if we want to support python 3.
Ok though.